src/GraphQL.Tests/ObjectExtensionsTest.cs

```diff
@@ -1,5 +1,6 @@
 namespace GraphQL.Tests
 {
+    [Collection("StaticTests")]
```

Comment:
As I understand, there is no need to capture all the tests from test classes to make them sequential - only ones that uses `UseCultures` but I'm OK to keep as is since performance drop is almost invisible.

Reply:
That’s what I was thinking as well